projects/kit/components/tabs/tab/tab.component.ts

```diff
@@ -29,6 +29,7 @@ import {TUI_TAB_EVENT, TUI_TAB_PROVIDERS} from './tab.providers';
     providers: TUI_TAB_PROVIDERS,
     host: {
         '($.data-mode.attr)': 'mode$',
+        '[style.--tui-tab-margin.px]': 'margin',
```

Comment:
it would be cool if it was observable so people can change the value dynamically